src/ergw_aaa_radius.erl

```diff
@@ -184,6 +205,14 @@ validate_option(Opt, Value)
   when (Opt =:= server_name orelse Opt =:= client_name)
        andalso is_binary(Value) ->
     Value;
+validate_option(timeout, Value)
+  when is_integer(Value) andalso Value > 0 ->
+    Value;
+validate_option(retries, Value)
+  when is_integer(Value) andalso Value > 0 ->
+    Value;
+validate_option(async, Value) when (Value =:= true) ->
```

Comment:
is_boolean/1 please, setting async to `false` is valid setting.

Reply:
done